e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -38,10 +35,38 @@ Scenario: Re-enrolling an agent
   When the agent is re-enrolled on the host
     And the "agent" process is "started" on the host
   Then the agent is listed in Fleet as online
-    And there is data in the index
+    And there is new data in the index from agent
 
 @revoke-token
 Scenario: Revoking the enrollment token for an agent
   Given an agent is deployed to Fleet
+    And the agent is un-enrolled
+    And the "agent" process is "stopped" on the host
   When the enrollment token is revoked
```

Comment:
I believe that we are going to need the ability to create tokens using APIs, so that we can revoke it without affecting the default one.

Reply:
discussed via chat - to confirm here, I agree - we should manage this better:

to create a new token (and they are config specific)
POST to /api/ingest_manager/fleet/enrollment-api-keys

with a POST body like:
{“name”:“test-token”,“config_id”:“99a28950-bae3-11ea-9dd9-f131eddc2919"}

the config_id is the id of the configuration and name is arbitrary name

to get the config_ids available, you can call a GET to /api/ingest_manager/agent_configs
  and walk the returned list, config_id is listed

Reply:
got it working!

Reply:
More context on this: we decided to have each scenario to prepare its state (i.e. enrollment token) so that we can have each scenario idempotent